lib/presentation/mixin/load_more_search_mixin.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:fluffychat/pages/new_private_chat/fetch_contacts_controller.dart';
+import 'package:fluffychat/presentation/model/search/presentation_search.dart';
+import 'package:flutter/material.dart';
+
+mixin LoadMoreSearchMixin {
+  final scrollController = ScrollController();
+
+  final haveMoreCountactsNotifier = ValueNotifier(true);
+
+  final lastContactIndexNotifier = ValueNotifier(0);
+
+  List<PresentationSearch> oldContactList = [];
+  List<PresentationSearch> oldContactAndRecentChatList = [];
+
+  void listenForScrollChanged({required FetchContactsController fetchContactsController}) {
+    scrollController.addListener(() {});
```

Comment:
what does it do ?

Reply:
I will update them by adding load more logic.

This ticket: https://github.com/linagora/twake-on-matrix/issues/317